cli/types/tests/cypress-tests.ts

```diff
@@ -29,19 +29,20 @@ namespace CypressAutomationTests {
 
 namespace CypressConfigTests {
   // getters
-  Cypress.config('baseUrl') // $ExpectType string | null
+  Cypress.config('baseUrl') // $ExpectType string | null | undefined
   Cypress.config().baseUrl // $ExpectType string | null
 
   // setters
   Cypress.config('baseUrl', '.') // $ExpectType void
-  Cypress.config({ e2e: { baseUrl: '.' }}) // $ExpectType void
-  Cypress.config({ e2e: { baseUrl: null }}) // $ExpectType void
-  Cypress.config({ e2e: { baseUrl: '.', }}) // $ExpectType void
+  Cypress.config({ e2e: { baseUrl: '.' }}) // $ExpectError
```

Comment:
Why is defining `e2e.baseUrl` now a type error?

Reply:
Cypress.config({ e2e: ...}) isn't actually valid:

<img width="662" alt="Screen Shot 2022-11-08 at 12 30 07 PM" src="https://user-images.githubusercontent.com/14099737/200677577-c73c1d87-1b88-482c-be42-a09f9b0b84aa.png">